src/httpurr/client.cljc

```diff
@@ -4,7 +4,7 @@
   (:refer-clojure :exclude [get])
   (:require [promesa.core :as p]
             [httpurr.protocols :as proto])
-  (:import goog.Uri))
+  #?(:clj (:import java.net.URL) :cljs (:import goog.Uri)))
```

Comment:
`java.net.URI`'s API is a nightmare, there is no way to convert a unescaped string into an URL wich `java.net.URL` does. I think we are fine.